src/environment.ts

```diff
@@ -232,7 +232,7 @@ export class Environment {
           })`;
         }
       } else {
-        if (name === "unescape") {
+        if (name === "unescape" || name === "safe") {
           unescaped = true;
```

Comment:
If we include a `continue` here, we no need to create a plugin with an empty function. `safe` can be considered a core feature of Vento, in the same way as `autoescape` option.